src/Components/ArticlePage.jsx

```diff
@@ -1,5 +1,25 @@
-function ArticlePage (){
-return <h1>hu</h1>
+import Card from 'react-bootstrap/Card'
+import {useParams} from 'react-router'
+import {Link} from 'react-router'
+
+function ArticlePage ({articles}){
+    const {article_id} = useParams();
+    const article = articles.find((article) => article.article_id.toString() === article_id);
```

Comment:
This may work but is not making use of your api at all here. How can you make use of your api to fetch a specific article id?